mission_control/src/mission_control.cpp

```diff
@@ -34,165 +34,102 @@
 
 // Original version: Christopher Scianna Christopher.Scianna@us.QinetiQ.com
 
+#include "mission_control/mission_control.h"
+
 #include <map>
 #include <string>
 
-#include "mission_control/mission_control.h"
-
 MissionControlNode::MissionControlNode(ros::NodeHandle& h) : node_handle(h)
 {
   heartbeat_sequence_id = 0;
-  last_id = -1;
   last_state = Mission::MissionState::READY;
   reportExecuteMissionStateRate = 1.0;  // every 1 second
   reportHeartbeatRate = 1.0;            // every 1 second
   m_current_mission_id = 0;
-  m_cur_mission = NULL;
   m_mission_id_counter = 0;
 
-  disable_abort = false;
   reportExecuteMissionStateRate = 1.0;
   reportHeartbeatRate = 1.0;
 
   // Get runtime parameters
-  node_handle.getParam("/mission_control_node/mission_path", mission_path);
-  node_handle.getParam("/mission_control_node/disable_abort", disable_abort);
-
   node_handle.getParam("/mission_control_node/report_execute_mission_state_rate",
                        reportExecuteMissionStateRate);
   node_handle.getParam("/mission_control_node/report_heart_beat_rate", reportHeartbeatRate);
 
-  ROS_INFO("mission path:[%s]", mission_path.c_str());
-  ROS_INFO("disable abort:[%s]", disable_abort ? "true" : "false");
-  ROS_INFO("report executemissionstate rate:[%lf]", reportExecuteMissionStateRate);
-  ROS_INFO("report heartbeat rate:[%lf]", reportHeartbeatRate);
+  ROS_INFO_STREAM("report executemissionstate rate: " << reportExecuteMissionStateRate);
+  ROS_INFO_STREAM("report heartbeat rate: " << reportHeartbeatRate);
 
   // Subscribe to all topics
-  sub_corrected_data = node_handle.subscribe("/pose/corrected_data", 1,
-                                             &MissionControlNode::correctedDataCallback, this);
   report_fault_sub = node_handle.subscribe("/health_monitor/report_fault", 1,
                                            &MissionControlNode::reportFaultCallback, this);
-
-  load_mission_sub = node_handle.subscribe("/mngr/load_mission", 1,
+  load_mission_sub = node_handle.subscribe("/mission_control_node/load_mission", 1,
                                            &MissionControlNode::loadMissionCallback, this);
-  execute_mission_sub = node_handle.subscribe("/mngr/execute_mission", 1,
+  execute_mission_sub = node_handle.subscribe("/mission_control_node/execute_mission", 1,
```

Comment:
@patapufate if I may, I think we should set things straight here and use private names instead e.g.:
```suggestion
  ros::NodeHandle pnh("~");
  execute_mission_sub = pnh.subscribe("execute_mission", 1,
```
Same everywhere else.

Reply:
Fixed